### ISSUE TITLE
doesn't work in older linux distros such as fedora 23

### ISSUE THREAD

**Issue by mailinglists35** [m59peacemaker/mupdf-appimage#2]
what are the requirements? isn't appimage supposed to contain everything?

````
/tmp/.mount_MuPDF-hkfifM/usr/bin/mupdf-gl: error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory
````

**Comment by m59peacemaker** [comment]
You are correct - the app image is supposed to have all the dependencies included. I have only used `mutool` in a few environments and it worked. I know very little about creating app images and apparently did not do a great job here. I probably won't have time to investigate this soon. Feel free to improve upon my work if you wish. I would be very grateful to hear of what you learn if so!
